app/controllers/v0/profile/connected_applications_controller.rb

```diff
@@ -49,19 +47,17 @@ def apps_from_grants
         payload = { icn: }
         url_with_params = "#{grant_url}?#{URI.encode_www_form(payload)}"
         headers = { apiKey: Settings.connected_apps_api.connected_apps.api_key }
-        response = RestClient::Request.execute(method: :get, url: url_with_params, headers:)
 
-        if response.code == 200
+        response = Faraday.get(url_with_params, {}, headers)
+        if response.status == 200
           parsed_response = JSON.parse(response.body)
           lhapps = parsed_response['apps']
           lhapps.each do |lh_app|
             app = build_apps_from_data(lh_app)
             (data ||= []) << app
           end
-          { 'data' => data }
         end
-      rescue
-        { data: [] }
+        data
```

Comment:
Again, looking at codeowners, this belongs to the `vfs-authenticated-experience-backend` team, so I'm not sure I'd refactor unless it's necessary  to get it working 🤷 

Also, does this change not affect how the controller action behaves? What's preventing a Faraday/Request error from being returned to the client if the intention is for them to get `{ data: [] }`?